docs/usage.md

```diff
@@ -164,12 +182,31 @@ This pipeline uses featureCounts to generate QC metrics based on [biotype](http:
 
 Please get in touch with us on the #rnaseq channel in the [nf-core Slack workspace](https://nf-co.re/join) if you are having problems or need any advice.
 
+### iGenomes (not recommended)
+
+If the `--genome` parameter is provided (e.g. `--genome GRCh37`) then the FASTA and GTF files (and existing indices) will be automatically obtained from AWS-iGenomes unless these have already been downloaded locally in the path specified by `--igenomes_base`.
+
+However this is no longer recommended because:
+
+- Gene annotations in iGenomes are extremely out of date. This can be particularly problematic for RNA-seq analysis, which relies on accurate gene annotation.
+- Some iGenomes references (e.g., GRCh38) point to annotation files that use gene symbols as the primary identifier. This can cause issues for downstream analysis, such as the nf-core [differential abundance](https://nf-co.re/differentialabundance) workflow where a conventional gene identifier distinct from symbol is expected.
+
+Notes:
+
+- As of v3.7 of the pipeline, if you are using a genome downloaded from AWS iGenomes and using `--aligner star_salmon` (default) the version of STAR to use for the alignment will be auto-detected (see [#808](https://github.com/nf-core/rnaseq/issues/808)).
+
 ## Running the pipeline
 
 The typical command for running the pipeline is as follows:
 
 ```bash
-nextflow run nf-core/rnaseq --input <SAMPLESHEET> --outdir <OUTDIR> --genome GRCh37 -profile docker
+nextflow run \
+    nf-core/rnaseq \
+    --input <SAMPLESHEET> \
+    --outdir <OUTDIR> \
+    --gtf <GTF> \
+    --fasta <GENOME FASTA> \
```

Comment:
don't we need `--igenomes_ignore --genome null` too?

Reply:
... actually, could I be more provocative and suggest we lose `--igenomes_ignore` in favour of a new `--load_igenomes`? So non-use of iGenomes becomes the default position (since it's what we're recommending)?

Reply:
(realise I've broken the tests, I'll fix things up if folk agree this is the right way to go)

Reply:
What does `--ignomes_ignore` do specifically?  I just run with with `--fasta` and `gtf` and don't specify `--genome`  at all (is that equivalent to `--igenomes null`?), and that works great (I do have `igenomes_base` in my profile). 
Also, is there anything in iGenomes (QC references, etc.?) that is being used by default, even when `--fasta` and `--gtf` are provided?  I suspect not, because when I don't specify the `--genome` I don't seem to be missing anything.  If that is in fact the case, I am totally fine with not using iGenomes by default.